data/data/bootstrap/bootstrap-in-place/opt/openshift/bootstrap-in-place/master-update.fcc

```diff
@@ -0,0 +1,66 @@
+variant: fcos
+version: 1.1.0
+ignition:
+  config:
+    merge:
+      - local: original-master.ign
+storage:
+  trees:
+    - local: kubernetes/bootstrap-configs
+      path: /etc/kubernetes/bootstrap-configs
+    - local: tls/
+      path: /etc/kubernetes/bootstrap-secrets
+    - local: etcd-bootstrap/bootstrap-manifests/secrets/
+      path: /etc/kubernetes/static-pod-resources/etcd-member
+    - local: etcd-data
+      path: /var/lib/etcd
+  files:
+    - path: /etc/kubernetes/bootstrap-secrets/kubeconfig
+      contents:
+        local: auth/kubeconfig-loopback
+    - path: /etc/kubernetes/static-pod-resources/etcd-member/ca.crt
+      contents:
+        local: tls/etcd-ca-bundle.crt
+    - path: /etc/kubernetes/manifests/etcd-pod.yaml
+      contents:
+        local: etcd-bootstrap/bootstrap-manifests/etcd-member-pod.yaml
+    - path: /etc/kubernetes/manifests/kube-apiserver-pod.yaml
+      contents:
+        local: bootstrap-manifests/kube-apiserver-pod.yaml
+    - path: /etc/kubernetes/manifests/kube-controller-manager-pod.yaml
+      contents:
+        local: bootstrap-manifests/kube-controller-manager-pod.yaml
+    - path: /etc/kubernetes/manifests/kube-scheduler-pod.yaml
+      contents:
+        local: bootstrap-manifests/kube-scheduler-pod.yaml
+    - path: /usr/local/bin/bootstrap-in-place-post-reboot.sh
+      contents:
+        local: bootstrap-in-place/bootstrap-in-place-post-reboot.sh
+      mode: 0555
+    - path: /var/log/log-bundle-bootstrap-in-place-pre-reboot.tar.gz
+      contents:
+        local: log-bundle-bootstrap-in-place-pre-reboot.tar.gz
+    - path: /usr/local/bin/installer-masters-gather.sh
+      contents:
+        local: bin/installer-masters-gather.sh
+      mode: 0555
+    - path: /usr/local/bin/installer-gather.sh
+      contents:
+        local: bin/installer-master-bootstrap-in-place-gather.sh
+      mode: 0555
+systemd:
+  units:
+    - name: bootstrap-in-place-post-reboot.service
```

Comment:
@eranco74 I believe the bootkube service no longer exists at this point. Does it make sense to land `bootstrap-in-place-post-reboot.service` as `bootkube` instead? The script could then echo a note about being a BiP workflow and it would show up at the top of the bootkube service logs. This would also remove the necessity of collecting this new service logs in the `installer-gather.sh`.

Reply:
Hmmm, I must say that I like it :smile: ,
This service serves the same purpose as bootkube.
Debugging is more straight forward (don't need to remember the new service name)
I'm just a bit concerned that it might be confusing, and less clear if the node is after reboot or before the reboot.
@romfreiman, @tsorya @omertuc your thoughts on this?

Reply:
I guess it can work. No strong objection from my side. Just have explicit logging there.